utils/oscap_docker_python/oscap_docker_util.py

```diff
@@ -178,19 +178,39 @@ def _cleanup_by_path(self, path):
 
 
 class OscapScan(object):
-    def __init__(self, tmp_dir=tempfile.gettempdir(), mnt_dir=os.getcwd(),
+    def __init__(self, tmp_dir=tempfile.gettempdir(), mnt_dir=None,
                  hours_old=2):
         self.tmp_dir = tmp_dir
         self.helper = OscapHelpers(tmp_dir)
         self.mnt_dir = mnt_dir
         self.hours_old = hours_old
 
+    def _get_mnt_dir(self):
+        '''
+        Get directory for mount container
+        '''
+        if self.mnt_dir is None:
+            return tempfile.mkdtemp()
+        else:
+            return self.mnt_dir
+
+    def _remove_mnt_dir(self, mnt_dir):
+        '''
+        Remove temporary directory if it was not passed by __init__
+        We don't want to remove directory what was created by used
+        '''
+        if self.mnt_dir != mnt_dir:
```

Comment:
I don't think we can do either of those in `maint-1.2`. It's public API after all.